Makefile

```diff
@@ -57,12 +57,14 @@ build/pyodide.asm.js: \
 		--exclude-file "*/tests/*"
 	# Strip out C++ symbols which all start __Z.
 	# There are 4821 of these and they have VERY VERY long names.
-	# Reduces size of pyodide.asm.js by a factor of 2.
-	# I messed around with striping more and could remove another 400kb or so
-	# but the regexes I got were generated.
-	# To show some stats on the symbols you can use the following:
-	# cat build/pyodide.asm.js | grep -ohE 'var _{0,5}.' | sort | uniq -c | sort -nr | head -n 20
 	sed -i -E 's/var __Z[^;]*;//g' build/pyodide.asm.js
+	sed -i '1i\
+		"use strict";\
+		let setImmediate = globalThis.setImmediate;\
+		let clearImmediate = globalThis.clearImmediate;\
```

Comment:
It would be good to help fixing this in emscripten. For instance https://github.com/emscripten-core/emscripten/issues/12568

Reply:
They might be worried about `globalThis`, but I guess they can always make a shim for it.

Reply:
It was OK in https://github.com/emscripten-core/emscripten/pull/8904, so maybe not.

Reply:
The other variables `baseName`, `fpcGOT`, `dyncallGOT`, `fpVal`, and `dcVal` have all been removed from upstream emscripten AFAICT so I think if that PR is merged this will likely be all resolved on upstream.